delivery/create-cloud-installation/README.md

```diff
@@ -0,0 +1,61 @@
+# create-cloud-installation
+
+This action is used to create a Mattermost cloud installation from Github Actions.
+
+### Usage
+
+```yaml
+jobs:
+  create-cloud-installation:
+    runs-on: <private-runners>
+    steps:
+      - uses: mattermost/actions/delivery/install-mattermost-cloud-@main
+        with:
+          version: 0.83.0
+          
+      - uses: mattermost/actions/delivery/create-cloud-installation@main
+        with:
+          dns: my-first-instance.test.mattermost.com
+          server: https://delivery-provisioner.test.mattermost.com
```

Comment:
Is the idea for this action to cater different provisioner server depending on which repo will consume or we will use the same in all places? If the latter, should this be defined within the action itself and no need to expose together with its corresponding `x-api-key`?

Reply:
This is an internal server in any case and the KEY will be a secret . In the future we might need to use different provisioner URLs so I do not want to hardcode it . 
We can keep the URL a secret too if that is the case, but again this is a public DNS name pointing on internal IPs